lib/os.js

```diff
@@ -144,19 +143,60 @@ function endianness() {
 }
 endianness[Symbol.toPrimitive] = () => kEndianness;
 
+function countOnes(n) {
+  let count = 0;
+  while (n !== 0) {
+    n = n & (n - 1);
+    count++;
+  }
+  return count;
+}
+
+function getCIDR({ address, netmask, family }) {
+  let ones = 0;
+  let split = '.';
+  let range = 10;
+  let groupLength = 8;
+  let hasZeros = false;
+
+  if (family === 'IPv6') {
+    split = ':';
+    range = 16;
+    groupLength = 16;
+  }
+
+  const parts = netmask.split(split);
+  for (const part of parts) {
```

Comment:
Since you are rewriting this for performance, is `for...of` as fast as the classic for loop now?

Reply:
Tracking bug: https://bugs.chromium.org/p/v8/issues/detail?id=8070